stdlib/public/core/Sort.swift

```diff
@@ -507,7 +507,7 @@ extension MutableCollection where Self: RandomAccessCollection {
       // If a child is bigger than the current node, swap them and continue sifting
       // down.
       if largest != i {
-        swapAt(idx, largest)
+        swapAt(i, largest)
```

Comment:
If there is no reason ever to use the original value `idx`, then the first line of the function body should be `var idx = idx`.